docs/content/manual/_incoming/start-vm-after-harvester-upgrade-complete.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Check can start VM after Harvester upgrade
+---
+
+* Related issues: 
+
+## Category: 
+* Harvester Upgrade
+
+## Verification Steps
+1. Prepare the previous stable Harvester release cluster
+1. Create image
+1. Enable Network and create VM
+1. Create seveal virtual machine
```

Comment:
should be "several"

Reply:
Appreciated for the check, just correct this typo.